setup.py

```diff
@@ -210,7 +212,10 @@ def run(self):
     if sys.platform != 'win32':
         cmake_command.append(source)
         self.spawn(cmake_command)
-        self.spawn(['make'])
+        if PARALLEL:
```

Comment:
Having a parallel build done through setup.py on unix is great.
At some point we should make this use the `-MP` option on Windows as well, but it's harder to do that without interfering with a user-specified `CMAKE_CXX_FLAGS`.